### PR TITLE
:bug: Linux builds from CI now work

### LINKED ISSUE #521: Fix Popen calls in linux

### PR DESCRIPTION
**Pull request by fabianvf**
- Added an ENV constant, with the pyinstaller python overrides removed
- All subprocess calls now use this

closes #521

### CODE REVIEW

**Review by jmontleon — APPROVED** [review]
ACK

File: kai/constants.py [hunk 1]
```diff
@@ -32,3 +33,22 @@
...
 PATH_TESTS = os.path.join(PATH_GIT_ROOT, "tests")
 PATH_TEST_DATA = pathlib.Path(os.path.join(PATH_GIT_ROOT, "tests/test_data"))
+
+
+def __clean_env() -> dict[str, Any]:
+    """
+    Returns the environment that should be used when calling out to a subprocess
+    """
+    env: dict[str, Any] = dict(os.environ)  # make a copy of the environment
+    lp_key = "LD_LIBRARY_PATH"  # for GNU/Linux and *BSD.
```
Comment (shawn-hurley) [inline kai/constants.py:43]: Should this only be run if we are in the pyinstaller-created binary?

Comment (jmontleon) [reply kai/constants.py:43]: Do you anticipate that this would cause a problem if not running in pyinstaller?

**Review by shawn-hurley — APPROVED** [review]

File: kai/constants.py (hunk 1 above)
Comment (fabianvf) [reply kai/constants.py:43]: In a sufficiently strange environment, probably

Comment (fabianvf) [reply kai/constants.py:43]: locally it works fine for me outside of the binary, I think we can leave it for now and revisit if someone reports an issue.

**Pull request merged**